lib/msal-browser/src/broker/BrokerMessage.ts

```diff
@@ -16,7 +16,7 @@ export abstract class BrokerMessage {
 
     static validateMessage(message: MessageEvent): MessageEvent|null {
         if (message.data && message.data.messageType) {
-            if (message.data.messageType in BrokerMessageType) {
+            if (Object.values(BrokerMessageType).indexOf(message.data.messageType) > -1) {
```

Comment:
Object.values not supported in all browsers

Reply:
done.